tests/unit/time-series-test.js

```diff
@@ -256,3 +256,31 @@ test('[Dynamic X Axis] Labels cannot be less specific than the maximum time spec
   ind2 = domainOrder.indexOf(this.subject().get('xAxisTimeInterval'));
   return assert.equal(ind >= ind2, true, 'The labels are not less specific than the max time specificity');
 });
+
+test('Tick Spacing', function(assert) {
+  var component = this.subject();
+  var tickSpacing = component.get('tickSpacing');
+
+  assert.expect(6);
+
+  assert.equal(component.get('_innerTickSpacingX'), tickSpacing,
+    'tickSpacingX should be equal to the tickSpacing if no default value is set');
+  assert.equal(component.get('_innerTickSpacingY'), tickSpacing,
+    'tickSpacingY should be equal to the tickSpacing if no default value is set');
+
+  Ember.run(function() {
```

Comment:
Because  in testing mode, auto-run is disabled so we have to put async code in a run loop